libs/designsystem/src/lib/components/calendar/calendar.component.spec.ts

```diff
@@ -0,0 +1,237 @@
+    spectator.setInput('disabledDates', ['1997-08-25', '1997-08-26'].map(localMidnightDate));
+    spectator.setInput('selectedDate', localMidnightDate('1997-08-29'));
+
+    const captured = captureDateChangeEvents();
+
+    clickDayOfMonth(26); // August 26st was explicitly disabled
+    expect(captured.event).toBeUndefined();
+  });
+
+  it('should always emit a dateChange event when clicking today if alwaysEnableToday is set to true', () => {
+    const today = localMidnightDate('1997-08-28');
+    spectator.setInput('disabledDates', [today]);
+    spectator.setInput('todayDate', today);
+    spectator.setInput('alwaysEnableToday', true);
+    spectator.setInput('selectedDate', localMidnightDate('1997-08-29'));
+
+    const captured = captureDateChangeEvents();
+
+    clickDayOfMonth(28); // August 28th is was disabled but we override
+    expect(captured.event).toEqual(today);
+  });
+
+  it('should emit dateChange event as UTC midnights when when timezone is set to UTC', () => {
+    spectator.setInput('timezone', 'UTC');
+    spectator.setInput('selectedDate', utcMidnightDate('1997-08-29'));
+
+    const captured = captureDateChangeEvents();
+
+    clickDayOfMonth(14);
+    expect(captured.event).toEqual(utcMidnightDate('1997-08-14'));
+  });
+
+  it('should be tolerant of date input (selectedDate, todayDate, minDate, maxDate, and disabledDates) as both UTC midnight and local time midnight', () => {
+    const localDate = localMidnightDate('1997-08-29');
+    const utcDate = utcMidnightDate('1997-08-29');
+
+    spectator.setInput('selectedDate', localDate);
+    expect(spectator.component.selectedDate).toEqual(localDate);
+    spectator.setInput('selectedDate', utcDate);
+    expect(spectator.component.selectedDate).toEqual(localDate);
+
+    spectator.setInput('todayDate', localDate);
+    expect(spectator.component.todayDate).toEqual(localDate);
+    spectator.setInput('todayDate', utcDate);
+    expect(spectator.component.todayDate).toEqual(localDate);
+
+    spectator.setInput('minDate', localDate);
+    expect(spectator.component.minDate).toEqual(localDate);
+    spectator.setInput('minDate', utcDate);
+    expect(spectator.component.minDate).toEqual(localDate);
+
+    spectator.setInput('maxDate', localDate);
+    expect(spectator.component.maxDate).toEqual(localDate);
+    spectator.setInput('maxDate', utcDate);
+    expect(spectator.component.maxDate).toEqual(localDate);
+
+    const localDates = ['1997-08-29', '1997-08-30'].map(localMidnightDate);
+    const utcDates = ['1997-08-29', '1997-08-30'].map(utcMidnightDate);
+
+    spectator.setInput('disabledDates', localDates);
+    expect(spectator.component.disabledDates).toEqual(localDates);
+    spectator.setInput('disabledDates', utcDates);
+    expect(spectator.component.disabledDates).toEqual(localDates);
+  });
+
+  // constants and utility functions
+
+  const SEL_NAV_BACK = '.previous-month-button';
```

Comment:
Consider using button placement as selector instead of unused css class:
```suggestion
  const SEL_NAV_BACK = '.header button:first-of-type';
```

Reply:
Yep, that's a better approach. I'll remove the CSS classes as well.